packages/features/auth/lib/next-auth-options.ts

```diff
@@ -439,6 +455,7 @@ export const AUTH_OPTIONS: AuthOptions = {
           impersonatedByUID: token.impersonatedByUID as number,
           belongsToActiveTeam: token?.belongsToActiveTeam as boolean,
           organizationId: token?.organizationId,
+          token: token?.isOrgAdmin,
```

Comment:
Not sure if we should return a token or a boolean here

Reply:
The property of the token is a boolean. Should I cast the type as boolean like what's done above?